single-line-diagram/single-line-diagram-core/pom.xml

```diff
@@ -91,5 +91,20 @@
             <artifactId>powsybl-tools-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-loadflow-api</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-open-loadflow</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why do you need those?

Reply:
I used those dependencies because they made it easy to get P, Q and I values displayed in tests. But I removed them and now I set those 3 values "by hand".

Reply:
You removed the use of loadflow in tests but you didn't remove the dependency